bridge/actions/ActionContext.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <atomic>
+namespace bridge
+{
+
+class Mixer;
+class MixerManager;
+struct StreamDescription;
+class RequestLogger;
+} // namespace bridge
+
+namespace transport
+{
+class SslDtls;
+} // namespace transport
+
+namespace bridge
+{
+struct ActionContext
+{
+    ActionContext(bridge::MixerManager& mixerManager, transport::SslDtls& sslDtls)
+        : _mixerManager(mixerManager),
+          _sslDtls(sslDtls),
+          _lastAutoRequestId(0)
+    {
+    }
+    bridge::MixerManager& _mixerManager;
```

Comment:
while refactoring. Should we expose these without _.  As they are not private to all the functions operating on this struct?

Reply:
Fixed.